app/controllers/appointment_summaries_controller.rb

```diff
@@ -15,6 +15,7 @@ def create
   def show
     appointment_summary = AppointmentSummary.find(params[:id])
     output_document = OutputDocument.new(appointment_summary)
+    OutputDocumentMailer.guidance_record(appointment_summary).deliver_later
```

Comment:
The issue of passing in `appointment_summary` versus `output_document` was something @gjvis and myself discussed briefly on Friday.

The mailer currently requires the `AppointmentSummary` as it contains the email address to send the mail to and when we get the copy for the email body it'll likely require personal details from this object as well.

But yeah, instantiating the document twice, in both the controller and the mailer is a thing to think about. Passing the generated document into the mailer along with the summary is probably the way forward.